test/unit/storage/keyvalue/JsonResourceStorage.test.ts

```diff
@@ -126,4 +127,24 @@ describe('A JsonResourceStorage', (): void => {
   it('can handle resources being deleted while iterating in the entries call.', async(): Promise<void> => {
     //
   });
```

Comment:
Completely irrelevant but just noticed this. Can you remove this test since it's empty anyway?